pages/index/_learn-more.html

```diff
@@ -53,7 +53,7 @@ <h2>Get access to the Infrastructure as Code Library.</h2>
     <h2>Deploy your entire tech stack.</h2>
     <p>
       Deploy your own tech stack by following our
-      <a href="/guides">Production Deployment Guides</a> or have Gruntwork
+      <a href="https://docs.gruntwork.io">Documentation</a> or have Gruntwork
```

Comment:
```suggestion
      <a href="https://docs.gruntwork.io/docs/guides/build-it-yourself/overview/">deployment guides</a>, or have Gruntwork
```

Reply:
We'd like to take this change, but since we can't commit the suggestion and the PR is from a fork, we'll merge this and then update this link in a separate PR.